metaswitch/common/alarms_parser.py

```diff
@@ -33,6 +33,12 @@
 
 # Valid severity levels - this should be kept in sync with the 
 # list in alarmdefinition.h in cpp-common
+# Alarms are stored in ITU Alarm Table using the severities below.
+# Alarm Model Table stores alarms according to their state. The
+# mapping between state and severity is described in RFC 3877
+# section 5.4
```

Comment:
Can we have a direct link here? "described at https://tools.ietf.org/html/rfc3877#section-5.4 under ituAlarmEntry" would be more useful.